### DISCUSSION TITLE
Restore Plex LXC

### DISCUSSION THREAD

**Discussion by zerofruchtshake** [tteck/Proxmox#1694] [Help Needed]
Hi, it it possible to backup a Plex LXC and then restore on a new server with newer hardware and have Hardware Acceleration working? Or should i do a clean install?

**Comment by tteck** [comment]
A backup and restore _should_ work fine.

**Reply by bobobeastie** [reply]
A couple of days ago I did a fresh proxmox install to go from 7 to 8, I restored a plex lxc that was created with your script a while ago which had functioning hardware encoding.  I have since restored from the backup again, because my install managed to become a fresh install somehow.  After both restores I have not seen hardware transcoding working.  I have looked through as many guides as I can find and can't find anything wrong.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/1056#discussioncomment-6458037

**Reply by bobobeastie** [reply]
https://github.com/tteck/Proxmox/discussions/1694#discussioncomment-6763875
Awesome you came through again.  The key for me, I think, was:

`crw-rw---- 1 root input 226, 128 Aug 18 17:52 renderD128`
needed to be:
`crw-rw---- 1 root render 226, 128 Aug 18 17:52 renderD128`

That was the only difference I could find.  Thank you so much.

 ...and likely adding these lines to the conf file:
`lxc.autodev: 1
lxc.cgroup2.devices.allow: a
lxc.cap.drop:`

**Comment by zerofruchtshake** [comment]
80% CPU usage when transcoding a single 4K HDR to 1080p high suggests HW acceleration not working, right? 😅
![Bildschirmfoto 2023-08-16 um 11 09 35](https://github.com/tteck/Proxmox/assets/44063194/1f0c0df4-4c16-4eef-aad0-51814275f5d9)

**Reply by ghost** [reply]
Hi, install tautulli then you can see if Plex is using HW acceleration, but with 80% CPU usage I guess not. When HW acceleration is active on my server the CPU usages is almost zero.

  
![Schermafbeelding 2023-08-16 om 20 56 53](https://github.com/tteck/Proxmox/assets/12942317/6ffa919d-0e10-400f-ba37-6e1af72d412f)
